crates/electrum/src/bdk_electrum_client.rs

```diff
@@ -242,11 +428,11 @@ impl ElectrumSyncResult {
     /// This requires additional calls to the Electrum server.
     pub fn with_confirmation_time_height_anchor(
         self,
-        client: &impl ElectrumApi,
+        client: &BdkElectrumClient<impl ElectrumApi>,
```

Comment:
❓ Not introduced by this PR, but why do we need the `client` param? Wouldn't we always use &self.inner? Same above for `full_scan()`.